File: web3rpc/rpc-specs/code-samples/javascript/personal/deriveAccount.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const url = "url";
  const path = "path";
  const pin = true;

  provider.personal
    .deriveAccount(url, path, { pin }, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/personal/deriveAccount.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

url = "url"
path = "path"
pin = True

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.derive_account(url, path, pin)

print(personal_response)
